challenge.md

```diff
@@ -8,36 +8,48 @@ Write your answers inside this file, where it's indicated by the comments.
 
 <!-- Answer Starts Here -->
 
+We'd use the git clone command, followed by an SSH URL cloned from GitHub.
+
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
 <!-- Answer Starts Here -->
 
+git diff
+
 <!-- Answer Ends Here -->
 
 3. Oops - it looks like there was a change that we forgot to include in our last commit. How can we revise that commit and fix things?
 
 <!-- Answer Starts Here -->
 
+git commit --amend
+
 <!-- Answer Ends Here -->
 
 4. It looks like there was a change on the original repo that we forked from. How could we grab those changes and incorporate them into our local repo?
 
 <!-- Answer Starts Here -->
 
+We'd need to pull from upstream and merge the changes from upstream into our version.
```

Comment:
That's correct - the command is `git pull upstream` (optionally, plus a branch).